src/lpython/semantics/python_ast_to_asr.cpp

```diff
@@ -3484,6 +3522,7 @@ class SymbolTableVisitor : public CommonVisitor<SymbolTableVisitor> {
             }
             bool ltypes, enum_py;
             set_module_symbol(msym, paths);
+            // std::cout<<"msym: "<<msym<<std::endl;
```

Comment:
Some debug print statements should be removed.

Reply:
Yes. Going to do that and clean up the git history.